client/src/routes/admin/adminGebruikerInzien/GebruikerInzien.tsx

```diff
@@ -1,9 +1,61 @@
+import React, {
+	useCallback
+} from 'react';
 import {
-	Box, Card, TextField, Typography 
+	Box,
+	Button,
+	Card,
+	Dialog,
+	DialogActions,
+	DialogContent,
+	DialogContentText,
+	DialogTitle,
+	FormControl,
+	InputLabel,
+	MenuItem,
+	Select,
+	Typography
 } from '@mui/material';
-import React from 'react';
+import API from '../../../api/apiRoutes';
+import {
+	Form 
+} from 'react-router-dom';
 
 function GebruikerInzien() {
+
+	const [users, setUsers] = React.useState([]);
+	const [selectedUser, setSelectedUser] = React.useState();
+	const [open, setOpen] = React.useState(false);
+
+	React.useEffect(() => {
+		async function getUsers() {
+			const userData = (await API('gebruikers').GetAll()).data;
+			setUsers(userData);
+		}
+
+		getUsers();
+	}, [setUsers]);
+
+	const usersList = users.length > 0
+		&& users.map((user: any, i: any) => {
+			return (
+				<MenuItem key={i} value={user}>{user.naam}</MenuItem>
```

Comment:
for the key add the user.id instead (small optimization but you can leave it like that for now)